hawkbit-repository/hawkbit-repository-test/src/main/java/org/eclipse/hawkbit/repository/test/util/PostgreSqlTestDatabase.java

```diff
@@ -8,61 +8,55 @@
  */
 package org.eclipse.hawkbit.repository.test.util;
 
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.PreparedStatement;
-import java.sql.SQLException;
-
 import org.apache.commons.lang3.RandomStringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.test.context.TestExecutionListener;
+import org.springframework.util.StringUtils;
 
 /**
  * A {@link TestExecutionListener} for creating and dropping MySql schemas if
  * tests are setup with MySql.
  */
 public class PostgreSqlTestDatabase extends AbstractSqlTestDatabase {
 
-    private static final Logger LOG = LoggerFactory.getLogger(PostgreSqlTestDatabase.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(PostgreSqlTestDatabase.class);
+    protected static final String POSTGRESQL_URI_PATTERN = "jdbc:postgresql://{host}:{port}/{path}?currentSchema={db}*";
```

Comment:
With `"jdbc:postgresql://{host}:{port}/{path}?currentSchema={db}*"` the currently configured default postgres URL (`spring.datasource.url=jdbc:postgresql://localhost:5432/hawkbit`) will not work. 
Therefore I would suggest to switch it to `"jdbc:postgresql://{host}:{port}/{database}*";`, which will allow both types.

Reply:
In addition, this connection parameter is optional and should not be enforced (https://jdbc.postgresql.org/documentation/head/connect.html)